src/modules/systemlib/param/param.c

```diff
@@ -1356,7 +1363,7 @@ uint32_t param_hash_check(void)
 
 	/* compute the CRC32 over all string param names and 4 byte values */
 	for (param_t param = 0; handle_in_range(param); param++) {
-		if (!param_used(param)) {
+		if (!param_used(param) && param_is_volatile(param)) {
```

Comment:
Shouldn't this be `if (!param_used(param) || param_is_volatile(param)) {`?

Reply:
Indeed!

Reply:
Followed up here: https://github.com/PX4/Firmware/commit/3d9cd5c0bcd6e8903a9e2d68b0835aba42dc9cdc